model/host/stats.go

```diff
@@ -110,6 +110,8 @@ func statsByDistroPipeline() []bson.M {
 	return []bson.M{
 		{
 			"$match": bson.M{
+				// don't count user-spawned hosts EVG-15232
+				StartedByKey: evergreen.User,
```

Comment:
once we have this do we still need [the check](https://github.com/evergreen-ci/evergreen/blob/dd3199cc8f9ee0d3316f8b1251a3d956a3b3ab83/model/host/stats.go#L116-L117) that it wan't started by a task? If it was started by a task wouldn't the StartedBy field be the task id?

Reply:
Yes, I think it would be.